packages/common/src/audius-query/AudiusQueryContext.ts

```diff
@@ -1,14 +1,15 @@
 import { createContext } from 'react'
 
 import { AudiusSdk } from '@audius/sdk'
-
+import { ReportToSentryArgs } from '../models'
 import type { AudiusAPIClient } from 'services/audius-api-client'
 import { AudiusBackend } from 'services/index'
 
 export type AudiusQueryContextType = {
   apiClient: AudiusAPIClient
   audiusSdk: () => Promise<AudiusSdk>
   audiusBackend: AudiusBackend
+  reportToSentry: (args: ReportToSentryArgs) => void
```

Comment:
Do we actually need this in the context if we're catching all errors in `CreateApi` anyway? It allows the endpoint's implementer to call report to sentry which is added utility anyway I suppose.

Reply:
Ooh good point, but yeah I just added it here because I couldn't figure out a better way to report to Sentry since we're in common